### ISSUE TITLE
Probability of hospitalisation given infection should be group specific but currently isn't

### ISSUE THREAD

**Issue by aezarebski** [nefeltellioglu/ethical_framework#22]
In the current set of parameters we have different hospitalisation rates for each of the groups (i.e. the parameters suggest there is a different rate of hospitalisation upon infection depending upon group, conditional upon infection). The code does not yet support this, but could be edited to do so by making the following edits:

1. The `BurdenParams` needs to be updated to have the group specific hospitalisation rates that we now have: https://github.com/nefeltellioglu/ethical_framework/blob/81d0753b314cc20196afa521055d040fbf4aa69e/ethics/model.py#L22
2. The `loss_terms` function in the `ethics.model` module needs to be updated: https://github.com/nefeltellioglu/ethical_framework/blob/81d0753b314cc20196afa521055d040fbf4aa69e/ethics/model.py#L964
3. The `create-grid-database.py` script will need to be updated to construct `BurdenParams` properly after it is updated.
4. One of the plotting scripts will also need to be updated, although this is in a function that should probably not be used anyway, it would be better to make use of a library function for this calculation.

Basically, this will all stem from editing the `BurdenParams` class to have the following definition:

```python
  @dataclass
  class BurdenParams:
      # Infection burden parameters
      prop_hosp_inf_1: float
      prop_hosp_inf_2: float
      days_hosp_inf_1: float
      days_hosp_inf_2: float

      # Protection from disease among (unprotected) vaccinated.
      # TODO Something about this seems a bit fishy to me, maybe we need
      # a clearer variable name. Is it a proportion???
      vacc_protection_from_disease_1: float
      vacc_protection_from_disease_2: float
  
      # Vaccination burden parameter
      prop_hosp_vacc_1: float
      prop_hosp_vacc_2: float
      days_hosp_vacc_1: float
      days_hosp_vacc_2: float
```
